File: RAI/Clase13/Python/Animal.py

class Animal:

    def __init__(self, nombre:str, familia:str, especie:str):
        self.nombre = nombre
        self.familia = familia
        self.especie = especie
    
    def hace_sonido(self):
        print(self.nombre + " hace sonido...")

    def come(self, comida:str):
        print(self.nombre + " come "+ comida)

File: RAI/Clase13/Java/Animal.java
//package RAI.Clase13.Java;

public class Animal {
    protected String nombre;
    protected String familia;
    protected String especie;

    public Animal(String nombre, String familia, String especie){
        this.nombre = nombre;
        this.familia = familia;
        this.especie = especie;
    }

    public void haceSonido(){
        System.out.println(this.nombre + " hace un sonido");
    }

    public void come(String comida){
        System.out.println(this.nombre + " come " + comida);
    }
}
